plugins/accessibility/index.js

```diff
@@ -17,12 +17,37 @@ var q = require('q'),
  *      }]
  *    }
  *
+ *    // Tenon.io:
+ *
+ *    //  Read about the Tenon.io settings and API requirements:
+ *    //  -http://tenon.io/documentation/overview.php
+ *    //  -http://tenon.io/documentation/understanding-request-parameters.php
+ *
+ *    exports.config = {
+ *      ...
+ *      plugins: [{
+ *        tenonIO: {
+ *          options: {
```

Comment:
Hmm - can we actually just pass the entire options object to tenon, and just modify the 'src'? So here say something like:

tenonIO: {
  options: {} // See http://tenon.io/documentation/understanding-request-parameters.php
                   // The src will be added by the test.